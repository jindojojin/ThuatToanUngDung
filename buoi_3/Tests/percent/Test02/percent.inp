20
42 8 35 1 14
60 5 19 9 7
13 5 6 6 70
42 8 2 2 46
56 3 28 7 6
52 5 3 4 36
13 3 22 7 55
40 50 0 0 10
2 9 10 3 76
5 5 5 5 80
4 2 3 4 87
34 5 22 2 37
15 5 15 5 60
20 20 20 20 20
40 0 40 0 20
60 0 0 0 40
9 7 41 3 40
15 9 27 6 43
1 10 11 1 77
27 2 34 9 28
