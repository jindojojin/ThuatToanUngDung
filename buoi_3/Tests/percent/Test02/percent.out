100
100
100
50
100
100
100
10
100
20
100
100
20
5
5
5
100
100
100
100
